src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md

```diff
@@ -142,10 +142,6 @@ We have fixed hundreds of issues in the Magento 2.3.4 core code.
 
 *  Upgrades no longer fail when deployments contain store information predefined in `app/etc/config.php`. Previously, MySQL locks occurred during the store import phase of upgrade, and Magento threw an error.
 
-<!--- MC-20135-->
-
-*  You can now successfully run `bin/magento setup:static-content:deploy` in developer mode. Previously, attempting to run `bin/magento setup:static-content:deploy` resulted in an error.
-
```

Comment:
I couldn't find a reference to `MC-20135` on the `2.4-develop` branch on the magento2 github repo.

But I believe this might be related to https://github.com/magento/magento2/issues/24678, which is already mentioned below in the `Command-line interface (CLI commands)` section.

Another reason for not mentioning this in the `Installation, upgrade, deployment` section, is because the issue never existed in 2.3.3, it got broken and fixed in between 2.3.3 and 2.3.4, so it's more of a non-issue.

I'm not 100% sure about this, maybe double check first!

Reply:
I'm following up on this with our PM and will remove if it is a regression bug -- thanks.